angular/projects/spark-angular/src/lib/components/sprk-alert/sprk-alert.stories.ts

```diff
@@ -11,9 +11,7 @@ export default {
     moduleMetadata({
       imports: [SprkAlertModule],
     }),
-    componentWrapperDecorator(
-      (story) => `<div class="sprk-o-Box">${story}</div>`,
-    ),
+    componentWrapperDecorator((story) => `<div>${story}</div>`),
```

Comment:
I don't think we need to wrap these stories in anything. Did you discover that div needs to be there for a reason?

Reply:
good point! I will investigate and if removing them is fine I will do that!